.floo

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please update your .gitignore file to include this file. This is not supposed to be in github.